standalone.md

```diff
@@ -54,7 +54,31 @@ The TestGrid configuration located at `gs://your-bucket/config` can be rendered
 - The service account `k8s-testgrid@appspot.gserviceaccount.com` must be given
 permission to read from these files.
 
+## Examples
+
+### Standalone Testgrid FE For Knative Prow
+
+#### Generate testgrid config
+
+Set up a prow job that generates testgrid config from prow configs with
+`configurator`. In knative scenario, the [configurator job example] writes the
+config to a GCS bucket `gs://knative-own-testgrid/config`.
+
+After this step, `testgrid.k8s.io/r/knative-own-testgrid` can already work, but
+there is no data yet.
+
+#### Load data
+
+Create the [deployment] of `Updater` and `Summarizer` in testgrid cluster (Note:
+this can be any cluster of your choice), make sure that both deployments
```

Comment:
I think this should be simply replaced with "your" or "a" kubernetes cluster. These are kubernetes deployments that can run anywhere, and for many users maybe shouldn't run in the k8s-community cluster.

Reply:
done